app/templates/index.html

```diff
@@ -4,7 +4,6 @@
 
 {% block top_header %}
 <header class="marketplace-homepage-heading">
-  {% include "toolkit/phase-banner.html" %}
```

Comment:
We can also remove the import for the corresponding scss file in application.scss

Reply:
unfortunately, the CSS is still required for the beta logo. As part of the removal of phase banner, I will separate these